05-deploying_shiny_app.Rmd

```diff
@@ -1,279 +1,239 @@
-2. In R Studio, checkout this copy (file -> new project -> version control -> git).   (TODO:  Remember to make sure the template has a Rproj that enables packrat)
-3. Work on your Shiny app, using the template `ui.R` and `server.R` files.
-4. Ensure that packrat is enabled and your package dependencies are snapshotted using `packrat::snapshot()`
-5. Set the desired access permissions in `deploy.json` (i.e. whether it should be available for DOM1, Quantum, or external).
-6. When you're ready to share it, in GitHub create a 'release' and it will deploy in a few minutes.
-7. Ask the Platform team to add and remove users from your app in the [#ap_admin_request](https://asdslack.slack.com/messages/CBLAGCQG6/) Slack channel.
+### Summary
 
-Step-by-step instructions are below.
+In order to deploy a Shiny app, you should follow these steps:
 
-## Step-by-step guide to deploying an app
+1.  Copy the app template
+2.  Create a new webapp
+3.  Clone the repository
+4.  Develop the app
+5.  Manage dependencies
+6.  Set access permissions
+7.  Create a release in GitHub
+8.  Deploy in Concourse
+9.  Add users to the app
+10. Access the app
 
-### Copy the template project into a new Github repository
+### Copy the app template
 
-1. Begin by making a **copy** of the R Shiny template project on Github: https://github.com/new/import.
+1.  Go to [github.com/new/import](https://github.com/new/import).
+2.  Enter `https://github.com/moj-analytical-services/rshiny-template` in the field titled 'Your old repository’s clone URL'.
+3.  Select `moj-analytical-services` as the owner of the new repository.
+4.  Add a name for the new repository.
+5.  Set the privacy of the new repository to private.
+6.  Select 'Begin import'.
 
-2. Enter `https://github.com/moj-analytical-services/rshiny-template` in the input box entitled 'your old repository’s clone URL:'
+### Create a new webapp
 
-3. Ensure the 'owner' of the new repository is 'moj-analytical-services' and choose a name for your repository:
+Standard users are not able to create new webapps or webapp data sources themselves.
 
-4. Make sure the repo is 'private' (this should be the default value):
+To create a new webapp or webapp data source, ask the Analytical Platform team on the [#ap_admin_request Slack channel](https://asdslack.slack.com/messages/CBLAGCQG6/) or by [email](mailto:analytical_platform@digital.justice.gov.uk), if you are a Quantum user.
 
-5. Click 'Begin import'
+You should provide the URL of the app's GitHub repository as well as any existing webapp data sources it should be connected to.
 
+### Clone the repository
 
-![](images/shiny/github_shiny_import.gif)
+1.  Navigate to the app's repository on GitHub.
+2.  Select 'Clone or download'.
+3.  Ensure that the dialogue says 'Clone with SSH'. If the dialogue says 'Clone with HTTPS' select 'Use SSH'.
```

Comment:
This is such a confusing area, so I suggest you add: `The URL should start with: "git@"`

Reply:
Changed in 6a5ffd9